### PR TITLE
[ADD][11.0] Adding module project_agile_workflow_transitions_by_task_type

### PR DESCRIPTION
**Pull request by pnajman-modoolar**
Hello,

Module ``project_agile_workflow_transitions_by_task_type`` module extends [project_agile](https://github.com/OCA/project-agile/pull/4) and introduces configuration of allowed task types for a specific task workflow transition.

Kind Regards,
Petar

### CODE REVIEW

File: project_agile_workflow_transitions_by_task_type/README.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline project_agile_workflow_transitions_by_task_type/README.rst]: Many remarks from here https://github.com/OCA/web/pull/881#pullrequestreview-99094143 applies in this PR (and probably most of the PR done today :smile: )

File: project_agile_workflow_transitions_by_task_type/models/project_workflow.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+class Workflow(models.Model):
+    _inherit = 'project.workflow'
+
+    def get_available_transitions(self, task, state):
+        transitions = super(Workflow, self).get_available_transitions(
+            task, state
+        )
+        task_types = frozenset([task.type_id.id])
+        return [
+            x
```
Comment (elicoidal) [inline project_agile_workflow_transitions_by_task_type/models/project_workflow.py]: I prefer to have meaningful names even for this kind of variable

**Review by elicoidal — CHANGES_REQUESTED** [review]
Can you add Unit tests and simple demo data?

File: project_agile_workflow_transitions_by_task_type/README.rst (hunk 1 above)
Comment (pnajman-modoolar) [reply project_agile_workflow_transitions_by_task_type/README.rst]: Fixed :)

**Pull request closed without merging**